NitroxPatcher/Patches/Dynamic/FootstepSounds_OnStep_Patch.cs

```diff
@@ -39,8 +44,16 @@ public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructio
                    new CodeInstruction(OpCodes.Ldloc_0),
                    new CodeInstruction(OpCodes.Ldarg_1),
                    new CodeInstruction(OpCodes.Call, Reflect.Method(() => CalculateVolume(default, default, default, default)))
-               )
-               .InstructionEnumeration();
+               ).MatchEndForward(new CodeMatch(OpCodes.Call, Reflect.Method((FMOD.Studio.EventInstance t) => t.release())),
+                                                               new CodeMatch(OpCodes.Pop))
+                                            .Advance(1)
+                                            .InsertAndAdvance(new CodeInstruction(OpCodes.Ldloc_0))
+                                            .Insert(new CodeInstruction(OpCodes.Call, Reflect.Method(() => SendFootstepPacket(default))))
+                                            .InstructionEnumeration();
```

Comment:
Formatting is again messed up
```suggestion
               ).MatchEndForward(
                   new CodeMatch(OpCodes.Call, Reflect.Method((FMOD.Studio.EventInstance t) => t.release())),
                   new CodeMatch(OpCodes.Pop)
               ).Advance(1)
               .InsertAndAdvance(new CodeInstruction(OpCodes.Ldloc_0))
               .Insert(new CodeInstruction(OpCodes.Call, Reflect.Method(() => SendFootstepPacket(default))))
               .InstructionEnumeration();
```

Reply:
I think formatting is fine, pretty sure it was included in the patch you sent me that I pushed